src/NBomber.Contracts/Metrics.fs

```diff
@@ -0,0 +1,25 @@
+﻿namespace NBomber.Contracts.Metrics
+
+type RequestCountThreshold =
+    | AllCount of (int -> bool)
+    | OkCount of (int -> bool)
+    | FailedCount of (int -> bool)
+    | FailedRate of (float -> bool)
+    | RPS of (float -> bool)
+
+type LatencyThreshold =
+    | Min of (float -> bool)
+    | Mean of (float -> bool)
+    | Max of (float -> bool)
+    | StdDev of (float -> bool)
+
+type LatencyPercentileThreshold =
+    | P50 of (float -> bool)
+    | P75 of (float -> bool)
+    | P95 of (float -> bool)
+    | P99 of (float -> bool)
+
+type Metric =
```

Comment:
It seems like this type is: RequestCountThreshold + LatencyThreshold + LatencyPercentileThreshold
Maybe it does make sense to name it just Threshold then?

Reply:
Makes great sense to me 👍

Reply:
Renamed this to `Threshold`.